src/configuration.js

```diff
@@ -68,28 +80,18 @@ export class Configuration {
 }
 
 function loadOptionsFromConfig(configDirectory) {
-  // If config path is not specified, look in root directory of project
-  // the first option to cosmiconfig.load can be absolute or relative
   const searchPath = configDirectory || './';
 
   const cosmic = cosmiconfig('graphql-schema-linter', {
     cache: false,
     sync: true,
   }).load(searchPath);
-  let options = {};
+
   if (cosmic) {
-    // Map config.rules -> `only` param
-    if (cosmic.config.rules) {
-      options.only = cosmic.config.rules;
-    }
+    return cosmic.config;
```

Comment:
That's a good point. It might be more prudent to start with a whitelist first. I'll make the change.